internal/guest/runtime/hcsv2/hostdata.go

```diff
@@ -0,0 +1,33 @@
+//go:build linux
+// +build linux
+
+package hcsv2
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/Microsoft/hcsshim/internal/guest/amdsev"
+)
+
+// ValidateHostData fetches SNP report (if applicable) and validates `hostData` against
+// HostData set at UVM launch.
+func ValidateHostData(hostData, reportData string) error {
```

Comment:
Is there a case where we would pass something for `reportData`? If not maybe we can just leave that param off.

Reply:
afaik SKR uses it when fetching SNP report, but as I'm not exporting this outside of internal, it'd make sense to omit it.